migrations/sql/V2021.03.10.09.45__update_now_application_identity_and_notice_of_work_view.sql

```diff
@@ -0,0 +1,68 @@
+/*set all existing applications to notice of work type */
+update now_application_identity
+set application_type_code = 'NOW';
+
+/* set type column as not null*/
+ALTER TABLE now_application_identity ALTER COLUMN application_type_code SET DEFAULT 'NOW'; 
+ALTER TABLE now_application_identity ALTER COLUMN application_type_code SET NOT NULL ;
+
+DROP VIEW IF EXISTS notice_of_work_view;
```

Comment:
Unless a future migration depends on changes made to the view I don't think there's a reason to update it here since the "R__" view files are always run last at the end. And it may also be nice to document the changes made to the view here.

Reply:
I was facing the issue when rerunnable was able to update view, so I had to do it explicitly